### PR TITLE
Editorial: (Exponentiation) Add NaN to exponent values whose behavior deviates from IEEE 754-2008

### PR DESCRIPTION
**Pull request by rwaldron**
This oversight was discovered and reported to me by James Clark (jjc@jclark.com).

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -13615,7 +13615,7 @@ <h1>Applying the `**` Operator</h1>
         <li>If _base_ &lt; 0 and _base_ is finite and _exponent_ is finite and _exponent_ is not an integer, the result is *NaN*.</li>
       </ul>
       <emu-note>
-        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
+        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity*, *-Infinity* or *NaN* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```
Comment (ljharb) [inline spec.html]: i'm confused; how did the first edition of ECMAScript specify a result for the exponentiation operation, which was added in the 7th edition?

Comment (michaelficarra) [reply spec.html]: `Math.exp`?

Comment (ljharb) [reply spec.html]: Ah, duh, i confused myself.

Comment (jmdyck) [reply spec.html]: Up until 28bdf079, that sentence was about `Math.pow(x, y)`, which did exist in the first edition.

**Comment by littledan** [comment]
For good measure, would it be OK to ask James Clark to sign [our IP form](https://tc39.github.io/agreements/contributor/)?

File: spec.html (hunk 1 above)
Comment (ljharb) [inline spec.html]:
```suggestion
        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity*, *-Infinity*, or *NaN* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```

**Comment by bakkot** [comment]
Per discussion in [this duplicate PR](https://github.com/tc39/ecma262/pull/1609#issuecomment-508279556), this isn't quite right. My suggested wording would be

> The result of base ** exponent when base is 1 or -1 and exponent is +Infinity or -Infinity, **or when base is 1 and exponent is NaN**, differs from IEEE 754-2019. The first edition of ECMAScript specified a result of NaN for ~this~**these** operation**s**, whereas later versions of IEEE 754-2019 specified 1. The historical ECMAScript behaviour is preserved for compatibility reasons.

cc @michaelficarra @apaprocki 

@rwaldron are you interested in updating this PR? If not, the editors will get to it in a week or two.

**Comment by bakkot** [comment]
OK, I've rebased this to resolve the merge conflict and pushed a commit tweaking the wording to be more precise.

File: spec.html [hunk 2]
```diff
@@ -1557,7 +1557,7 @@ <h1>Number::exponentiate ( _base_, _exponent_ )</h1>
             <li>If _base_ &lt; 0 and _base_ is finite and _exponent_ is finite and _exponent_ is not an integer, the result is *NaN*.</li>
           </ul>
           <emu-note>
-            <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity* differs from IEEE 754-2019. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2019 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
+            <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity*, or when base is *1* and exponent is *NaN*, differs from IEEE 754-2019. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2019 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```
Comment (syg) [inline spec.html:1560]:
```suggestion
            <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity*, or when _base_ is *1* and _exponent_ is *NaN*, differs from IEEE 754-2019. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2019 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```

Comment (bakkot) [reply spec.html:1560]: Done.

**Pull request merged**